### PR TITLE
qbs: Update to 1.19.1

### PR DESCRIPTION
**Pull request by ABBAPOH**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 11.4 20F71 x86_64
Xcode 12.5 12E262

###### Verification 
Have you

- [+] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [+] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [+] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [+] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [+] checked your Portfile with `port lint`?
- [-] tried existing tests with `sudo port test`?
- [+] tried a full install with `sudo port -vst install`?
- [+] tested basic functionality of all binary files?

### CODE REVIEW

File: devel/qbs/Portfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ PortGroup           qmake5 1.0
 PortGroup           compiler_blacklist_versions 1.0
 
 name                qbs
-version             1.18.2
+version             1.19.1
 revision            1
```
Comment (reneeotten) [inline devel/qbs/Portfile:9]: when updating to a new version the "revision" field should reset to zero

**Review by reneeotten — CHANGES_REQUESTED** [review]

File: devel/qbs/Portfile (hunk 1 above)
Comment (ABBAPOH) [reply devel/qbs/Portfile:9]: done

**Comment by ABBAPOH** [comment]
Hm, looks like a network issue?

**Comment by reneeotten** [comment]
thanks for the change! 

> Hm, looks like a network issue?

let's see - I restarted the jobs

**Review by reneeotten — APPROVED** [review]

**Pull request merged**